modules/installation-configuration-parameters.adoc

```diff
@@ -1882,6 +1882,140 @@ with an Azure cluster.
 ====
 endif::azure[]
 
+
+ifdef::agent[]
+[id="installation-configuration-parameters-additional-bare_{context}"]
+== Additional bare metal configuration parameters for the Agent-based Installer
+
+Additional bare metal installation configuration parameters for the Agent-based Installer are described in the following table:
+
+.Additional bare metal parameters
+[cols=".^2l,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|platform:
+  baremetal:
+    clusterProvisioningIP:
+|The IP address within the cluster where the provisioning services run.
+Defaults to the third IP address of the provisioning subnet.
+For example, `172.22.0.3` or `2620:52:0:1307::3`.
+|IPv4 or IPv6 address.
+
+|platform:
+  baremetal:
+    provisioningNetwork:
+|The `provisioningNetwork` configuration setting determines whether the cluster uses the provisioning network.
+If it does, the configuration setting also determines if the cluster manages the network.
+
+`Managed`: Default. Set this parameter to `Managed` to fully manage the provisioning network, including DHCP, TFTP, and so on.
+
+`Disabled`: Set this parameter to `Disabled` to disable the requirement for a provisioning network.
+When set to `Disabled`, you must only use virtual media based provisioning, or bring up the cluster using the assisted installer.
```

Comment:
I'm not sure the `bring up the cluster using the assisted installer `snippet is needed. Isn't this section only used for the agent-based installer which always uses the assisted installer?
The BMC credentials can still be set when using the assisted installer during the agent installation but they won't be used by the assisted installer. They will only be used after the cluster is installed, and then only if a user attempts a BMC operation like power-up/down.